website/docs/linker.md

```diff
@@ -0,0 +1,28 @@
+Specifies the linker.
+
+```lua
+linker("value")
+```
+
+### Parameters ###
+
+`value` is a string value.
+
+By default, `Default` and `LLD` are allowed, but toolsets or modules canprovide support for other linkers.
```

Comment:
This isn't consistent with other documented APIs, which just have a list of the values available in core.